### PR TITLE
Move sounds to FMOD

### PR DESCRIPTION
**Pull request by Govorunb**
This will:
- Reduce Unity asset build times by a bunch (FMOD Studio actually does builds correctly btw)
- Obsolete FMODSoundCollection (sad, I know) which is one fewer workaround we have to maintain
- Reduce total asset sizes by 5-10% since we can remove duplicate sounds + FMOD builds are just better

Come to the FMOD side, we have parameters, effects, mixing, and ---> *hot reload* <---
I fixed an issue entirely in FMOD Studio, pressed F7, and tested the fix without having to restart Subnautica. it's so good

### CODE REVIEW

**Comment by Alexejhero** [comment]
what am i even supposed to review here

**Comment by Govorunb** [comment]
yeah it's a bit much sorry
it should only show the actually relevant changes if you [skip the first commit](https://github.com/Alexejhero/Neurosama-Subnautica-Mod/pull/56/files/2a8c34be2f818c59ddfa8e4ac54d2740037933c7..bab5847f68615ff914f87c40cc642681718a6d12) and deselect `xml` and `meta` from the file filter

**Comment by Govorunb** [comment]
do you care if i just merge this? i've tested the sounds pretty extensively

i mean just look at this isn't it tantalising
image

File: .gitattributes [hunk 1]
```diff
@@ -3,4 +3,5 @@ SCHIZO/Resources/AssetBundles/* merge=ours
 *.asset text eol=lf
 *.prefab text eol=lf
 *.meta text eol=lf
-*.asmdef text eol=lf
\ No newline at end of file
+*.asmdef text eol=lf
+FMOD Project/**/*.xml text eol=lf
```
Comment (Alexejhero) [inline .gitattributes]: this doesnt work

**Review by Alexejhero — CHANGES_REQUESTED** [review]

**Review by Alexejhero — APPROVED** [review]

**Pull request merged**